src/styles/components/multiple-form.less

```diff
@@ -92,6 +92,21 @@ base/forms.less
 
   .multiple-form-modal {
 
+    .form-row-element {
+
+      &.checkbox {
+        &:extend(.align-self-flex-end);
+
+        &.column-1 {
```

Comment:
We need this because if a checkbox is in a row by itself, we don't want it to be aligned to the end of the flex container.